stargate_network/index.md

```diff
@@ -4,68 +4,58 @@ nav_order: 70
-2. Stargate Generation - It is preferable for the Stargate to be of a newer generation. As such, the 3rd Generation Pegasus Stargates will be chosen over the 2nd Generation Milky Way Stargates and Milky Way Stargates will be chosen over the 1st Generation Universe Stargates.
-3. Number of times used - If one Stargate has been used more times than another, it is very likely that travelers will wish to continue using it over the less used Stargate.
+## Preferred Stargate
+When a Stargate dials a Solar System, the Stargate Network will scan through the list of all available Stargates within given Solar System, and choose one to recieve the connection. It does so according to the three priorities it has, ordered in decrease of their importance. Higher priority will override lower priority:
+1. Presence of a DHD - Due to high importance of any traveler having the ability to return, this priority is the top one.
+2. Stargate Generation - Stargate Network will choose higher generation Stargates over lower ones, as higher generation most likely means they are more reliable and easy to use.
+3. Total connection count - The Network will prioritise Stargates that have a higher number of total connections, because that means the given Stargate must be more reliable as it is used much more than others.
 
 ## Getting an Address
-To get an Address of a Dimension, simply use one of the following commands:
-* `/sgjourney stargateNetwork address` to get the 7-chevron Address of a Dimension in the current Galaxy.
-* `/sgjourney stargateNetwork extragalacticAddress` to get the 8-chevron Address of a Dimension anywhere.
-* Find a **Naturally Generated** Cartouche.
-> Cartouches have Cartouche Tables similar to Chests with Loot Tables. You don't expect Chests you crafted yourself to have any Loot and just like that, Cartouches you place yourself won't have any interesting Addresses and will default to the Addresss of the Dimensions you've placed them in.
-* Give yourself a Cartouche with a Cartouche Table through commands, for example: `/give @p sjgourney:sandstone_cartouche{BlockEntityTag:{AddressTable:"sgjourney:cartouche_buried_gate"}}`
-> You can take a look at the [Cartouche Tables](https://github.com/Povstalec/StargateJourney/tree/main/src/main/resources/data/sgjourney/sgjourney/address_table) currently in the mod
+At the time of writing this part of the wiki, there are currently no ways to obtain specific 7 and 8-Symbol addresses in survival without using commands, but be aware that they are planned and will eventually be added.
+To get an address of a Dimension, there exists two commands:
+* `/sgjourney stargateNetwork address` is used to get a 7-Symbol address of a Dimension in the current Galaxy.
+* `/sgjourney stargateNetwork extragalacticAddress` is used to get a 8-Symbol address of a Dimension.
+In survival, you can find a **Naturally Generated** Cartouche on planets like Abydos and Chulak. They contain a 7-Symbol address of a random Solar System.
```

Comment:
add a blank line to separate the paragraph from the bullet list

[preview](https://github.com/lukaskabc/StargateJourney/blob/0dfc4988d9790f558b627fb48925c47c1b4925ab/stargate_network/index.md#getting-an-address)